packages/node_modules/@webex/internal-plugin-wdm/test/integration/spec/webex.js

```diff
@@ -5,33 +5,45 @@
 import '@webex/internal-plugin-wdm';
 
 import {assert} from '@webex/test-helper-chai';
-import sinon from 'sinon';
 import WebexCore from '@webex/webex-core';
 import testUsers from '@webex/test-helper-test-users';
 
-describe('plugin-wdm', function () {
-  this.timeout(30000);
+describe('plugin-wdm', () => {
```

Comment:
This file has been updated to test that the `@webex/internal-plugin-device` plugin mounts correctly when utilizing `@webex/internal-plugin-wdm`.